electrum/wallet.py

```diff
@@ -2395,8 +2395,8 @@ def export_request(self, x: Invoice) -> Dict[str, Any]:
             d['URI'] = self.get_request_URI(x)
             # if request was paid onchain, add relevant fields
             # note: addr is reused when getting paid on LN! so we check for that.
-            is_paid, conf, tx_hashes = self._is_onchain_invoice_paid(x)
-            if is_paid and (not self.lnworker or self.lnworker.get_invoice_status(x) != PR_PAID):
+            _, conf, tx_hashes = self._is_onchain_invoice_paid(x)
+            if not self.lnworker or self.lnworker.get_invoice_status(x) != PR_PAID:
```

Comment:
Well, this was not really a bug, more like an arbitrary design choice. But I guess we can change it if you want. Do it in a separate PR please.